Project.toml

```
@@ -1,6 +1,6 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.17.0"
+version = "0.18.0"
```

```suggestion
version = "0.17.1"
```

This is a nonbreaking change, which means that pre-1.0.0, it bumps the final digit